### ISSUE TITLE
Feature: Formatter

### ISSUE THREAD

**Issue by HoLLy-HaCKeR** [a5huynh/vscode-ron#5]
There currently don't seem to be any RON formatters for vscode. It would be very convenient to have, especially if you work with large RON structures like autogenerated text from Debug traits.

**Comment by a5huynh** [comment]
@HoLLy-HaCKeR excellent idea, I'll do a little research to see how difficult that would be to add

**Comment by eldyer** [comment]
There is an existing formatter for RON, ronfmt: https://github.com/afilip1/ronfmt

Currently comments aren't supported, but @afilip1 seems to be working on it. It would be awesome if this would be integrated into vscode-ron.

**Comment by sp-luciano-chinke** [comment]
I'm really interested with this. I would like to contribute if you are not currently working on this.

**Comment by a5huynh** [comment]
@sp-luciano-chinke Absolutely, go for it! I haven't done anything outside of some preliminary research into [how to integrate a formatter into the plugin](https://github.com/jrieken/vscode-formatter-sample). 

Keep us updated w/ your progress 😄

**Comment by damccull** [comment]
Definitely something I'd love to see. I just found out about ron files so I'm looking for all the tools still. Too bad rust or rust-analyzer don't have them by default. Thanks for making this project.
